image/image.go

```diff
@@ -168,17 +165,44 @@ func MockTrusted(mockTrusted []asserts.Assertion) (restore func()) {
 	}
 }
 
+func makeLabel(now time.Time) string {
+	return now.UTC().Format("20060102")
+}
+
 func setupSeed(tsto *ToolingStore, model *asserts.Model, opts *Options) error {
 	if model.Classic() != opts.Classic {
 		return fmt.Errorf("internal error: classic model but classic mode not set")
 	}
 
-	// sanity check target
-	if osutil.FileExists(dirs.SnapStateFileUnder(opts.RootDir)) {
-		return fmt.Errorf("cannot prepare seed over existing system or an already booted image, detected state file %s", dirs.SnapStateFileUnder(opts.RootDir))
-	}
-	if snaps, _ := filepath.Glob(filepath.Join(dirs.SnapBlobDirUnder(opts.RootDir), "*.snap")); len(snaps) > 0 {
-		return fmt.Errorf("need an empty snap dir in rootdir, got: %v", snaps)
+	core20 := model.Grade() != asserts.ModelGradeUnset
+	var rootDir string
+	var bootRootDir string
+	var seedDir string
+	var label string
+	if !core20 {
+		// Core 16/28,  writing for the writeable partion
```

Comment:
typo 
```suggestion
		// Core 16/18,  writing for the writeable partion
```

unless we have already a UC28?!?!?!